src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -592,22 +592,22 @@ else if( posHelper(model.getDinghy(), mycoord )){
     private static Coord AIHardFire(BattleshipModel model) {
         Random rand = new Random(System.currentTimeMillis());
         Coord mycoord;
-        //If the previious shot was not a hit
+        //If the previious shot was not a hit, shoot randomly
         do {
             mycoord = new Coord(rand.nextInt(boardHeight+1), rand.nextInt(boardWidth+1));
 
         } while (!checkValidShot(model, mycoord));//while the shot has already been done
 
         //If it was a hit
-
+        //Shot the areas that are around the preious shot because that is likely to be another hit
 
 
         return mycoord;
     }
 
     private static Coord AIEasyFire(BattleshipModel model) {
        Coord mycoord;
-
+        //Must be changed, some pattern for firing must be created

```

Comment:
I think the best way to do this is to just have the AI fire from left to right, top to bottom. It would be easy to implement with 2 for loops and it would be easy enough to play around.